scripts/utils/trading_strategy_helpers.js

```diff
@@ -246,10 +246,18 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
         const lowerLimit = lowestLimit * Math.pow(stepSizeAsMultiplier, bracketIndex)
         const upperLimit = lowerLimit * stepSizeAsMultiplier
 
-        const [upperSellAmount, upperBuyAmount] = calculateBuyAndSellAmountsFromPrice(upperLimit, stableToken, targetToken)
+        const [upperSellAmount, upperBuyAmount] = getUnlimitedOrderAmounts(
+          upperLimit,
+          targetToken.decimals,
+          stableToken.decimals
+        )
         // While the first bracket-order trades standard_token against target_token, the second bracket-order trades
         // target_token against standard_token. Hence the buyAmounts and sellAmounts are switched in the next line.
-        const [lowerBuyAmount, lowerSellAmount] = calculateBuyAndSellAmountsFromPrice(lowerLimit, stableToken, targetToken)
+        const [lowerBuyAmount, lowerSellAmount] = getUnlimitedOrderAmounts(
+          lowerLimit,
+          targetToken.decimals,
+          stableToken.decimals
+        )
```

Comment:
Not sure this this was pointed out, but you will see that the `targetToken` and `stableToken` were switched here when the new function was introduced. Was this intended or is this a fatal mistake?

Reply:
It's intended: the function `getUnlimitedOrderAmounts` puts target token first and then stable token, while `calculateBuyAndSellAmountsFromPrice` puts stable first and then target. I did this because we use target first then stable in most (but not all!) functions in this repo.